src/main/java/it/pagopa/interop/probing/eservice/operations/util/OffsetLimitPageable.java

```diff
@@ -34,27 +46,33 @@ public Sort getSort() {
 
 	@Override
 	public Pageable next() {
-		return null;
+		return new OffsetLimitPageable((int) getOffset() + getPageSize(), getPageSize(), getSort());
+	}
+
+	public OffsetLimitPageable previous() {
+		return hasPrevious() ? new OffsetLimitPageable((int) getOffset() - getPageSize(), getPageSize(), getSort())
```

Comment:
since we can't control the requests made by the client, we need to verify that 
`getOffset() - getPageSize()` >= 0, return 0 otherwise

Reply:
ok